examples/api/test/widgets/raw_menu_anchor/raw_menu_anchor.3_test.dart

```diff
@@ -0,0 +1,151 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' as ui;
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/raw_menu_anchor/raw_menu_anchor.3.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+String getPanelText(int i, AnimationStatus status) => 'Panel $i:\n${status.name}';
+
+Future<TestGesture> hoverOver(WidgetTester tester, Offset location) async {
+  final TestGesture gesture = await tester.createGesture(kind: ui.PointerDeviceKind.mouse);
+  addTearDown(gesture.removePointer);
+  await gesture.moveTo(location);
+  return gesture;
+}
+
+void main() {
+  testWidgets('Root menu opens when anchor button is pressed', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.RawMenuAnchorSubmenuAnimationApp());
+
+    final Finder button = find.byType(FilledButton);
+    await tester.tap(button);
+    await tester.pump();
+
+    final Finder panel =
+        find
+            .ancestor(of: find.textContaining('Submenu 0'), matching: find.byType(ExcludeFocus))
+            .first;
+
+    expect(
+      tester.getRect(panel),
+      rectMoreOrLessEquals(const ui.Rect.fromLTRB(347.8, 324.0, 524.8, 324.0), epsilon: 0.1),
+    );
+
+    await tester.pump(const Duration(milliseconds: 100));
+
+    expect(
+      tester.getRect(panel),
+      rectMoreOrLessEquals(const ui.Rect.fromLTRB(347.8, 324.0, 524.8, 499.7), epsilon: 0.1),
+    );
+
+    await tester.pump(const Duration(milliseconds: 101));
+    expect(
+      tester.getRect(panel),
+      rectMoreOrLessEquals(const ui.Rect.fromLTRB(347.8, 324.0, 524.8, 516.0), epsilon: 0.1),
+    );
+
+    expect(find.textContaining('Submenu'), findsNWidgets(4));
+  });
+
+  testWidgets('Hover traversal opens one submenu at a time', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.RawMenuAnchorSubmenuAnimationApp());
+
+    // Open root menu
```

Comment:
```suggestion
    // Open root menu.
```

Same for all the comments in this file.

Reply:
Done! Also added periods to two other instances. Thanks Bruno.